app/views/components/_publication-header.html.erb

```diff
@@ -0,0 +1,9 @@
+<header class="app-c-publication-header" id="contents">
```

Comment:
Should that ID still be there?

Reply:
Good spot, I'll look into it. Thanks 👍

Reply:
@fofr It's being used as an anchor for back to top link at the moment, see this page: https://government-frontend-pr-499.herokuapp.com/government/publications/budget-2016-documents/budget-2016 . 

I guess that should really be moved out of the component itself, maybe into a wrapper element within the template?

Reply:
Other pages link to the contents container:
https://www.gov.uk/guidance/hmrc-tools-and-calculators#contents